src/storage-controller/src/lib.rs

```diff
@@ -745,7 +759,14 @@ where
                                     output_index,
                                     storage_metadata: (),
                                 },
-                            )
+                            );
+
+                            // Cache the ingestion's cluster ID for the
+                            // ingestion export. This way we always have a
+                            // record of it, even if the ingestion's collection
+                            // description disappears.
+                            collection_state.collection_metadata.cluster_id =
+                                Some(ingestion_desc.instance_id);
```

Comment:
This is asserting that an ingestion is a dependency at `create_collections` time (but not necessarily at any other time) right? In any case, I don't think we should call this "cache"-ing, its just filling in the value later than other types of collections

Reply:
Makes sense